network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -737,7 +731,6 @@ def get_banner(self):
         MultiFieldPanel(
           [
             FieldPanel('hero_headline'),
-            FieldPanel('hero_story_description'),
             FieldRowPanel([
```

Comment:
```suggestion
            
```

This section is:
```python
            FieldRowPanel([
              FieldPanel('hero_button_text'),
              FieldPanel('hero_button_url'),
            ],
```
We don't need double nested FieldPanels here because it adds funky spacing between the admin fields (image at the bottom of this), I think this is a good PR to remove that. 

Suggestion:
```python
            ...
            FieldPanel('hero_headline'),
            FieldPanel('hero_button_text'),
            FieldPanel('hero_button_url'),
            ...
```

![image](https://user-images.githubusercontent.com/4743971/89789347-b48d2d00-dadd-11ea-9985-f62d8ef36691.png)

Reply:
Good call!